libraries/nprlib/nprlib_defs.mtlx

```diff
@@ -39,4 +39,18 @@
     <output name="out" type="float" default="0.0" />
   </nodedef>
 
+  <!--
+    Node: <gooch>
```

Comment:
The name of this node would benefit from more detail, and one option that comes to mind is `gooch_npr`.  If we went with this naming convention, then future NPR nodes that incorporate a lighting computation would similarly have the `npr` suffix, creating a parallel with `bsdf` in the physically based shading library.

Reply:
Another option for this node name would be `gooch_shade`, which suggests the presence of a lighting computation without implying a physically based light integration of the type found in the PBR library.

The name `gooch_shade` would also echo the common term for [Gooch Shading](https://en.wikipedia.org/wiki/Gooch_shading) in the industry, which might help to clarify its purpose.

Reply:
gooch_shade sounds reasonable. I'll update it.